backend/signature/signature-core/src/test/java/com/epam/indigoeln/signature/SignatureTest.java

```diff
@@ -32,7 +32,7 @@
 @TestMethodOrder(MethodOrderer.OrderAnnotation.class)
 @TestSecurity(user = "john")
 @JwtSecurity(claims = {@Claim(key = "given_name", value = "John"), @Claim(key = "family_name", value = "Doe")})
-class SignatureTest {
+class SignatureTest extends BaseTest {
```

Comment:
Right now the test fails because of some database inconsistencies

We can either fix it in scope of this PR, or disable the test for now, and continue with tests (and functionality) in next PR

I'd suggest we merge this one, but if you prefer we can continue here

If we are going to merge this one, please make sure the build is not failing: `gradlew build` or `./gradlew build`. Ping me if there are questions about other build error or something